bazel/linux/bundles.bzl

```diff
@@ -121,11 +121,7 @@ def _kunit_bundle(ctx):
     mods = get_compatible(ctx, ki.arch, ki.package, ctx.attr.module)
     alldeps = expand_deps(ctx, mods, ctx.attr.depth)
 
-    commands = [
-        # modprobe does not work correctly without /sys
-        "mount -t sysfs sysfs /sys || echo 1>&2 'Could not mount sys - modprobe may not work properly'",
-    ]
-
+    commands = []
```

Comment:
removing this without other changes will break existing tests and uses? What is the problem with the extra sysfs mount?

Can we first add the mount in users, and then remove the mount from enkit?

Reply:
this is shared with non kunit targets - eg, all kernel_qemu_run and qemu_test targets use bundles and generated code.

If any of them requires modprobe, this will break them.

Reply:
ummm, no.  This function is called `def _kunit_bundle(ctx):` -- it is only for kunit.  why would anything but kunit call this function?

this change does not affect any kunit tests.

Reply:
sorry, my bad - should have looked more carefully at the filename.